### ISSUE TITLE
support processing stdin

### ISSUE THREAD

**Issue by jonassmedegaard** [bensadeh/tailspin#24]
tailspin claims to be a drop-in replacement for tail -f, yet it does not support data from stdin.

This works: "echo foobar | tail -f"

This fails: "echo foobar | tailspin -f"

The more interesting use case is this: "journalctl -f | tailspin -f"

**Comment by bensadeh** [comment]
Hi Jonas,

I've added support for reading from stdin in c181bdbcae51dd0869e3ad36c50c54cbfbcb9ec8.

It should also work with the --follow flag as well, however if the user interrupts the follow mode (with ctrl + c), it currently wont pick up any new entries if the user tries to resume tailing with F again.

**Comment by bensadeh** [comment]
Fixed in `1.1.0`. Let me know if it is working like you expect.

**Comment by jonassmedegaard** [comment]
Works pretty much like I expected, yes. Thanks!

**Comment by jonassmedegaard** [comment]
As for less +F not picking up, perhaps it is related to a trick I learned from fellow Debian developer Jakub Wilk: If you pipe a command output to less, then the setup can get confused about which part should get the interrupt signals when interacting with it.

This gets confused:
```
journalctl -ef | less
```

whereas this works properly:
```
env --ignore-signal=INT journalctl -ef | less
```

**Comment by bensadeh** [comment]
Thank you for the tip. It is indeed an interrupt issue.

To simplify things and to shield the user from having to know about and deal with interrupt signals, I am experimenting with a new flag called `--tail-command`. It can be used like so:

```terminal
spin --tail-command 'command to be tailed'
```

From here, the user can interrupt and resume the tailing of anything the command prints to `stdout`. tailspin will listen to anything the commands prints to stdout for the duration of the main tailspin program.

**Comment by bensadeh** [comment]
The `--tail-command` flag is included in `1.2.1`. Might be good idea to include a section on tailing the stdout of another command.
